native/membrane_video_compositor/src/scene.rs

```diff
@@ -22,7 +24,8 @@ pub enum Node {
     Transformation {
         resolution: TextureOutputResolution,
         previous: Arc<Node>,
-        transformation: ImplementationPlaceholder,
+        // TODO: When we're ready, this should be swapped for `DecodedCustomStructElixirPacket`
+        transformation: CustomStructElixirPacket,
```

Comment:
Perhaps we could think of a better name than `Transformation.transformation`.

Reply:
I renamed it to `params`, same as in the layout